boards/arm/stm32h747i_disco/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright 2023 BrainCo Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Add custom linker section to relocate framebuffers to PSRAM
+zephyr_linker_sources_ifdef(CONFIG_LV_Z_VBD_CUSTOM_SECTION
```

Comment:
Is there any way to make this not board specific ?

Reply:
I'm not familiar with link script.  linker should get a board specified segment to store the lvgl_buf section.